.grit/patterns/js/no_implicit_coercion.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Disallow shorthand type conversions
+tags: [good-practice]
+---
+This matches the [eslint rule](https://eslint.org/docs/latest/rules/no-implicit-coercion).
```

Comment:
There's a lot more logic from the eslint rule that would be needed to make this correct. It's probably not useful tbh.

Reply:
@morgante  I found it very useful for readability. I ran it on an internal project +200 matches, it went smoothly. can you give an example ?

Reply:
If you look in the [eslint source](https://github.com/eslint/eslint/blob/main/lib/rules/no-implicit-coercion.js) it does more checks to ensure things like booleans aren't doubly cast. If you want to take the time to mirror that logic, I don't object - but most people will probably still just use eslint or biome.

Reply:
@morgante I actually I just noticed that I took the branch from the other feature (no implicit coercion). I'll split the PR. 

I will check the eslint source, but probably gonna look for another rule